ribs/emitters/opt/_cma_es.py

```diff
@@ -252,31 +255,30 @@ def _calc_strat_params(self, num_parents):
 
     @staticmethod
     @nb.jit(nopython=True)
-    def _calc_cov_update(cov, c1a, cmu, c1, pc, sigma, rank_mu_update):
+    def _calc_cov_update(cov, c1a, cmu, c1, pc, sigma, rank_mu_update, weights):
         """Calculates covariance matrix update."""
         rank_one_update = c1 * np.outer(pc, pc)
-        return (cov * (1 - c1a - cmu) + rank_one_update * c1 +
+        return (cov * (1 - c1a - cmu * np.sum(weights)) + rank_one_update * c1 +
```

Comment:
Did you get this from my DQD-RL code? I think we left it out because weights usually sum to 1, but we added it back to be more accurate to the CMA-ES code.